pyccl/nl_pt/ept.py

```diff
@@ -0,0 +1,700 @@
+import numpy as np
+import warnings
+from ..pk2d import Pk2D
+from ..base import CCLAutoreprObject, unlock_instance
+from ..errors import CCLWarning
+
+
+class EulerianPTCalculator(CCLAutoreprObject):
```

Comment:
Can we rename this to just `EulerianPT` which is much simpler and clear? It sets up Eulerian PT!
I am going to do the same with `HMCalculator`: rename it to `HaloModel` because it sets up a halo model, and it's much clearer.

Reply:
We had this discussion, so resolving.